ios/SketchView/SketchView.m

```diff
@@ -52,13 +52,9 @@ -(void)setToolType:(SketchToolType) toolType
     }
 }
 
--(void)setToolColor:(NSMutableDictionary *)rgba
+-(void)setToolColor:(UIColor *)rgba
 {
-    float r = [[rgba objectForKey:@"r"] floatValue];
-    float g = [[rgba objectForKey:@"g"] floatValue];
-    float b = [[rgba objectForKey:@"b"] floatValue];
-    float a = [[rgba objectForKey:@"a"] floatValue];
-    [(PenSketchTool *)penTool setToolColor:[UIColor colorWithRed:r/255.0 green:g/255.0 blue:b/255.0 alpha:a]];
+    [(PenSketchTool *)penTool setToolColor:rgba];
```

Comment:
I would leave the current `selectedTool` prop the way it is - you're always going to need to specify an ID, and this keeps backwards compatibility.  I would just add two new props, `toolColor` like what we have now and `toolThickness` so that if someone just wants the defaults of `black` and `5` they don't have to worry about it